views/interface.js

```diff
@@ -1,10 +1,26 @@
 document.addEventListener("DOMContentLoaded", function() {
 
-var storyComponent = new StoryComponent(story);
+	var api = "http://content.guardianapis.com/search?api-key=test"
 
- renderStory = function(storyComponent) {
- 		storyComponent.render();
- }
+	function getApi(api) {
+		var xmlHttp = new XMLHttpRequest();
+		xmlHttp.open("GET", api, false);
+		xmlHttp.send(null);
+		return xmlHttp.responseText;
+	}
 
- renderStory(storyComponent);
+	var apiString = getApi(api);
+	var apiJson = JSON.parse(apiString);
```

Comment:
Do we need JSON? We found that the results hash is already an object so doesn't need to be parsed.. may be different for Guardian api